.github/workflows/git-release.yml

```diff
@@ -0,0 +1,26 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Git Release
+
+on:
+  push:
+    tags:
+    - "v[0-9]+.[0-9]+.[0-9]+"
+  workflow_dispatch:
+  
+jobs:
+  Git_Release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
either needs the download artifacts task using the current sha and targeting the other workflow, or needs to build things itself. Suggestion https://github.com/marketplace/actions/download-workflow-artifact